### PR TITLE
Basic metrics

### PR DESCRIPTION
**Pull request by theminecoder**

### CODE REVIEW

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.health.HealthCheckRegistry;
+
+/**
+ * @author theminecoder
+ * @version 1.0
+ */
+public final class MetricsManager {
+
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: [File contains tab characters (this is the first instance).](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l11)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	private MetricsManager() {
+	}
+	
+	public static HealthCheckRegistry getHealthCheckRegistry() {
+		return healthCheckRegistry;
+	}
+	
+	public static MetricRegistry getMetricRegistry() {
+		return metricRegistry;
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: ['method def' child have incorrect indentation level 16, expected level should be 8.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l23)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import com.codahale.metrics.health.HealthCheckRegistry;
+
+/**
+ * @author theminecoder
+ * @version 1.0
+ */
+public final class MetricsManager {
+	
+	private static MetricRegistry metricRegistry = new MetricRegistry();
+	private static HealthCheckRegistry healthCheckRegistry = new HealthCheckRegistry();
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: ['member def modifier' have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l13)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+	private MetricsManager() {
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: ['ctor def modifier' have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l15)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+	}
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: ['ctor def rcurly' have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l16)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java (hunk 4 above)
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: [Missing a Javadoc comment.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l15)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+	
+	public static HealthCheckRegistry getHealthCheckRegistry() {
+		return healthCheckRegistry;
+	}
+	
+	public static MetricRegistry getMetricRegistry() {
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: ['method def modifier' have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l22)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+public final class MetricsManager {
+	
+	private static MetricRegistry metricRegistry = new MetricRegistry();
+	private static HealthCheckRegistry healthCheckRegistry = new HealthCheckRegistry();
+
+	private MetricsManager() {
+	}
+	
+	public static HealthCheckRegistry getHealthCheckRegistry() {
+		return healthCheckRegistry;
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: ['method def' child have incorrect indentation level 16, expected level should be 8.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l19)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.health.HealthCheckRegistry;
+
+/**
+ * @author theminecoder
+ * @version 1.0
+ */
+public final class MetricsManager {
+	
+	private static MetricRegistry metricRegistry = new MetricRegistry();
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: ['member def modifier' have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l12)

File: src/main/java/org/crafttogether/metrics/MetricsManager.java [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+	private static HealthCheckRegistry healthCheckRegistry = new HealthCheckRegistry();
+
+	private MetricsManager() {
+	}
+	
+	public static HealthCheckRegistry getHealthCheckRegistry() {
```
Comment (theminecoder) [inline src/main/java/org/crafttogether/metrics/MetricsManager.java]: Codacy Issue found: [Missing a Javadoc comment.](https://www.codacy.com/app/CraftTogether/CraftCore/file/2528729607/issues/source?bid=3277702&fileBranchId=3282827#l18)

**Comment by erosemberg** [comment]
no pls

**Pull request closed without merging**